### ISSUE TITLE
Translations in Transifex for v16 not fully synced

### ISSUE THREAD

**Issue by priojk** [Dolibarr/dolibarr#22021]
### Bug

I am checking the final translations for v16 for German (de_DE) and noticed that in general it's up to date, but some translations, which are present in Transifex, are still missing. 

One example: The key EX_PAR was originally translated as "Parkingspesen  Firmenauto", which I changed to "Parkgebühren Firmenwagen" around three months ago, according to the Transifex history. But in GitHub as of today, there is still the old translation.

Nothing really critical, but it happened already in the past that some translations were missing from the sync. I can just guess, but this example is located in the resource 'trips' on Transifex. Possibly, if no source code in this area got changed/updated, maybe the translations do not get fetched? Or something else is just slightly off?

**Comment by ksar-ksar** [comment]
Hello,

Normally, Sync has been done yesterday : https://github.com/Dolibarr/dolibarr/commit/044d2baca1585751958e9bc4a8b979a6a5073e78

But I do not see this modification : https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/langs/de_DE/trips.lang#L55

Can you check in Transifex ?

**Comment by priojk** [comment]
You are right, the key still has the old translation in GitHub:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/10772984/189432351-3f77c8d5-4caf-49d5-82cc-67f765c5a694.png">


I just checked this in Transifex. The modification has been made 3 months ago, so it definitely should have been synched in the meantime. There definitely seems to be some bug in the synching process.

<img width="392" alt="image" src="https://user-images.githubusercontent.com/10772984/189432852-a697c905-6131-49bf-91ee-251f6a5db602.png">

**Comment by gschratzer** [comment]
The last update from this lang-file was on on Jun 3
![image](https://user-images.githubusercontent.com/3923753/193567867-b2bdaa01-d842-43ca-a77c-4e635acf166c.png)
in version 16 and develop branch

**Comment by priojk** [comment]
Hi Guido, thanks for looking into this.

So the question is, why the changed translation has not been included in this Transifex sync from September:

<img width="589" alt="image" src="https://user-images.githubusercontent.com/10772984/193676159-38cef179-fb87-4e23-848f-469619bb99e2.png">

There must be some cause, why some files don't get included into the synchronisation, while most files do. Unfortunately, I do not know enough about the involved tooling, to answer this...

**Comment by gschratzer** [comment]
I think there is no monthly update when I look at the history, there changed only a few lines last year. 
It looks like the transferred file is not from the current transifex because the old content was transferred again.

![image](https://user-images.githubusercontent.com/3923753/193771061-73c0116f-4f09-451a-a2de-7c67248f81ff.png)

Maybe we should load the current status into the V 16 via pull request, and hope that the syncronization works again.

**Comment by ksar-ksar** [comment]
Hello,

I think, only @eldy can have a look on the script he uses to transfer the transfiex files into dolibarr.

**Comment by gschratzer** [comment]
in the version 17 it should be solved )) 
@priojk if you are agree, please close the issue
![image](https://user-images.githubusercontent.com/3923753/215999225-6191ba3a-fb9a-4d1b-b54c-cbd57e0016a8.png)

**Comment by priojk** [comment]
The provided example seems indeed to be resolved. Just recently I came across another translation, which apparently did not get synched correctly from Transifex, but I cannot remember the details (did no create a note). 

What I found when I did local synch with Transifex: The process depends on the timestamp of the local file. Everything is fine with a git clone (original timestamps). But once I do some copying of files, the old files get recent timestamps and are excluded from the synch. Maybe a hint for the root cause.

Will close the issue for now and open another one, if I find another example. Anyways, many thanks for looking into this.
